src/backend/oracle-data-api/src/main/resources/occm-openapi-spec.yaml

```diff
@@ -0,0 +1,615 @@
+openapi: 3.0.0
+info:
+  description: This is an Oracle REST Data Services (ORDS) service for the Online Court Case Management solution.
+  title: Online Court Case Management
+  version: 1.0.0
+servers:
+  - description: local
+    url: 'https://ords-0198bb-dev.apps.silver.devops.gov.bc.ca'
+paths:
+  /v1/ping:
```

Comment:
Do we really need to include "v1" for specific endpoint paths. Wouldn't it be better to keep it as part of the base URL so it will be much easier to update if we upgrade the version in the feature?